lib/app_profiler.rb

```diff
@@ -173,7 +173,7 @@ def backend
     end
 
     def vernier_supported?
-      RUBY_VERSION >= "3.2.1"
+      defined?(AppProfiler::Backend::VernierBackend.name) && RUBY_VERSION >= "3.2.1"
```

Comment:
These conditions should probably be reversed. The version check is likely lighter than the defined? check.

Reply:
I was thinking we should prioritize the one that's more likely to fail. Wdyt?